scalableminds/webknossos
# Use toast key to update existing toasts

In the recent Antd update, a change was included that allows to update existing notifications. This PR uses this functionality to update the notification that indicates that the move value changed, instead of rapidly closing and creating it, which looked kinda weird.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a tracing, change the move value in orthogonal and flight/oblique mode using g/h. The notification should be updated smoothly.

------
